hibernate-core/src/main/java/org/hibernate/bytecode/spi/ClassTransformer.java

```diff
@@ -35,5 +37,5 @@ byte[] transform(
 			String className,
 			Class<?> classBeingRedefined,
 			ProtectionDomain protectionDomain,
-			byte[] classfileBuffer);
+			byte[] classfileBuffer)  throws TransformerException;
```

Comment:
This is a change to an SPI method but should be inline with the [Jakarta 3.1 spec](https://jakarta.ee/specifications/persistence/3.1/apidocs/jakarta.persistence/jakarta/persistence/spi/classtransformer) of this interface.

Reply:
Yeah, this is fine